addons/docs/src/lib/docgen/extractDocgenProps.ts

```diff
@@ -33,9 +33,10 @@ export const extractComponentSectionArray = (docgenSection: any) => {
   const typeSystem = getTypeSystem(docgenSection[0]);
   const createPropDef = getPropDefFactory(typeSystem);
 
-  return docgenSection
-    .map((item: any) => extractProp(item.name, item, typeSystem, createPropDef))
-    .filter(Boolean);
+  return docgenSection.map((item: any) => {
+    const sanitizedItem = { ...item, value: item.elements };
```

Comment:
I think it should only do this if `item.elements && !item.value`.

I'm guessing that `react-docgen` and/or `react-docgen-typescript` fills in `item.value` and this would override it.

Reply:
Sure, I'll put that check in there. Just to get my head wrapped around this: I thought this code is only used by Vue since a few lines further down, there's this:

```
 // vue-docgen-api has diverged from react-docgen and returns an array
  return Array.isArray(docgenSection)
    ? extractComponentSectionArray(docgenSection)
    : extractComponentSectionObject(docgenSection);
};
```

Reply:
Right you are! False alarm!